sigs.yaml

```diff
@@ -862,7 +862,7 @@ sigs:
       control system reasonably consume?
 
       For more details about our objectives please review our
-      [Scaling And Performance Goals](https://github.com/kubernetes/community/blob/master/sig-scalability/goals.md)
+      [Scaling And Performance Goals](https://git.k8s.io/community/sig-scalability/goals.md)
```

Comment:
`[Scaling And Performance Goals](/sig-scalability/goals.md)`

Reply:
This one is the source for the link above in the readme. Because the relative links are only rendered in Markdown, I figured this should stay a fully qualified URL.